Yoast/javascript
# Add sentence beginnings assessment for Arabic

## Summary


This PR can be summarized in the following changelog entry:

* [yoastseo] Adds the consecutive sentences assessment for Arabic.
* [Yoast SEO] Adds the consecutive sentences assessment for Arabic.

## Test instructions

This PR can be tested by following these steps:

Test the following in WP SEO:
* Set the site language to Arabic.
* Write three or more Arabic sentences...
    * that all begin with the same written-out number, like `سبعة` ("seven")
       * E.g.  `.سبعة صوتا مقابل خمسة ضد، وامتناع واحد. سبعة صمامات مختلفة يتم إنتاجها في المبنى. سبعة متزوجين, واثنان في علاقة مع بعضهما` 
    * that all begin with the same demonstrative pronoun, like `هذا` ('this').
       * E.g. `.هذا الكهف مسكون. هذا التطبيق مكسور. هذا التطبيق مكسور` 
* In each of the above cases, the sentence beginning readability assessment should read: `Consecutive sentences: There is enough variety in your sentences. That's great!`
* Write three or more Arabic sentences that begin with the same word.
  * E.g. `.مرحبا ، اهلا وسهلا بكم. مرحبا كيف حالك؟ مرحبا هل انت في المنزل. مرحبا لجميع الوافدين الجدد`
* The sentence beginning readability assessment should read: `Consecutive sentences: The text contains 4 consecutive sentences starting with the same word. Try to mix things up!`

Test instructions for Lingo team:
* Run yarn test and make sure all current specs pass
* Run the Content analysis app and set the language to Arabic 
* Use test instructions from above.

## UI changes

## Quality assurance


Fixes https://yoast.atlassian.net/jira/software/projects/LIN/boards/4?selectedIssue=LIN-519
